kremlib/kremlib.h

```diff
@@ -370,9 +345,11 @@ typedef const char *Prims_string;
   }))
 #define be64toh(x) (htobe64((x)))
 
-#endif
+/* ... fail */
+#else
```

Comment:
Unrelated to this PR. I generally find it very helpful to put some comment to `#else`, `#endif`, and `#elif` explaining what it belongs to. When there's no indentation these constructs are always a little hard to parse imho.